HeliPort/Appearance/StatusMenu.swift

```diff
@@ -491,6 +495,16 @@ final class StatusMenu: NSMenu, NSMenuDelegate {
                 self.phyModeItem.title = NSLocalizedString("    PHY Mode: ", comment: "") + phyMode
                 self.mcsIndexItem.title = NSLocalizedString("    MCS Index: ", comment: "") + mcsIndex
                 self.nssItem.title = NSLocalizedString("    NSS: ", comment: "") + nss
+                if let wifiItemView = self.networkItemList[0].view as? WifiMenuItemView {
+                    wifiItemView.visible = connected
+                    if connected {
+                        wifiItemView.networkInfo = NetworkInfo(
```

Comment:
We could also set `isNetworkListEmpty` to false if there is a connection in case if there was no scanned items during this time in order to show the line separator. Refer to previous image under the connected item.